http-client/src/main/java/no/nav/familie/http/config/RestTemplateBuilderBean.kt

```diff
@@ -1,18 +1,27 @@
 package no.nav.familie.http.config
 
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.boot.web.client.RestTemplateBuilder
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.context.annotation.Import
+import org.springframework.context.annotation.Primary
 
 @Suppress("SpringFacetCodeInspection")
 @Configuration
 @Import(NaisProxyCustomizer::class)
 class RestTemplateBuilderBean {
 
     @Bean
+    @Primary
+    @ConditionalOnProperty("no.nav.security.jwt.issuer.azuread.proxyurl")
     fun restTemplateBuilder(iNaisProxyCustomizer: INaisProxyCustomizer): RestTemplateBuilder {
         return RestTemplateBuilder(iNaisProxyCustomizer)
     }
-}
 
+    @Bean
```

Comment:
Hvorfor trenger vi å ha RestTemplateBuilder-bean fra ett felles-bibliotek? 
Det gjør att autoconfig av resttemplate ikke virker.

Reply:
Hva er det som ikke fungerer?

Reply:
Ettersom noen applikasjoner bruker den andre så finnes det ju en risiko att de får "annet oppsett" hvis denne ikke er definert her, sånn att den burde vel kanskje med for å unngå andre problemer. 

Men då tenker jeg att att man kanskje burde bruke `ConditionalOnProperty` på begge beans, (matchIfMissing=true/false) i stedet for å bruke `@Primary`